### PR TITLE
fix(ui): Add explicit PatternFly style for links in classic pages

### PR DESCRIPTION
**Pull request by pedrottimark**
### Description

Follow up after addition of links to open side panel as solution to accessibility issue lack of keyboard access.

In contrast to that batch of separate contributions by container, this has a rare large number of changed files to make reduce the chance of inconsistencies in style changes.

### Problem

Presentation of links in classic tables is inconsistent with PatternFly tables.

### Analysis

1. `TableCellLink` component renders links with Tailwind classes.

### Solution

1. Edit cypress/integration/configmanagement/secrets.test.js file.
    * Replace `data-testid` attribute in selector corresponding to delete `testid` prop from `TableCellLink` component.

2. Edit cypress/integration/vulnmanagement/VulnerabilityManagement.selectors.js file.
    * Delete selector which also had `data-testid` attribute, but only for obsolete policies tests.

3. Edit cypress/integration/vulnmanagement/dashboardToEntityPage.test.js file.
    * Replace `div` with `a` element in selector corresponding to replacement of `onClick` event handler with ordinary `a` element.

4. Edit cypress/integration/vulnmanagement/entitypages.test.js file.
    * Delete obsolete tests for policies, because policies has been removed from Vulnerability Management.

5. Edit src/Components/FixableCVECount/FixableCVECount.js file.
    * Replace classic colors with PatternFly colors for link and text.
    * Also remove `text-sm` because information is both important and short.

6. Rename and edit src/Components/TableCellLink/TableCellLink.tsx file.
    * Rewrite in TypeScript and delete unneeded props.

7. Edit src/Components/visuals/SunburstDetailSection.js file.
    * Comment out unneeded props to prevent disruptive change to indentation.
    * Replace classic colors with PatternFly colors for link and text.

8. Edit src/Components/workflow/TableCountLink/TableCountLink.tsx
    * Delete unneeded `testid` prop.
    * Delete obsolete PropTypes.

9. Edit src/Containers/Compliance/widgets/ComplianceByStandard.js file.
    * Replace classic colors with PatternFly colors for link and text.

10. Edit src/Containers/Compliance/widgets/HorizontalBarChart.js file.
    * Replace classic colors (and underline) with PatternFly colors for link and text.

11. Edit src/Containers/ConfigManagement/Dashboard/widgets/ComplianceByControls.js file.
    * Replace classic colors with PatternFly colors for link and text.

12. Move and edit src/Containers/ConfigManagement/Dashboard/widgets/Lollipop.js file.
    * Replace classic colors with PatternFly colors for link.

13. Edit src/Containers/ConfigManagement/Dashboard/widgets/PolicyViolationsBySeverity.js file.
    * Replace classic colors with PatternFly colors for link and text.

14. Edit src/Containers/ConfigManagement/Dashboard/widgets/SecretsMostUsedAcrossDeployments.js file.
    * Replace classic colors with PatternFly colors for link and text.
    * Render link text at normal size.

15. Edit src/Containers/ConfigManagement/List/Secrets.js file.
    * Delete rare `testid` prop of `TextCellLink` element.

16. Edit src/Containers/VulnMgmt/widgets/CvesByCvssScore.js file.
    * Replace classic colors with PatternFly colors for link and text.

17. Edit src/Containers/VulnMgmt/widgets/LabeledBarGraph.js file.
    * Replace classic colors with PatternFly colors for link.

18. Move and edit src/Containers/VulnMgmt/widgets/NumberedGrid.js file.
    * Delete unused prop.
    * Replace classic colors with PatternFly colors for link and text.
    * Replace `div` and `onClick` prop with `Link` element.

19. Move and edit src/Containers/VulnMgmt/widgets/NumberedList.js file
    * Replace classic colors with PatternFly colors for link and text.

20. Edit src/css/trumps.css file.
    * Add style rules for `a` element in classic widget body and table cell.

### User-facing documentation

- [x] update is not needed
- [x] documentation PR is not needed

### Testing

- [ ] inspected CI results

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [x] modified existing tests

#### How I validated my change

1. `yarn lint` in ui/apps/platform
2. `yarn build` in ui/apps/platform and then compute branch - master for the following
    * `wc build/static/css/*.css`
        main.css -100 = 935423 - 935253
        total 170 = 1925616 - 1925446
    * `ls -al build/static/css/*.css | wc`
        files 0 = 44 - 44
    * `wc build/static/js/*.js`
        main.js 4 = 4615484 - 4615480
        total -1527 = 11698766 - 11700293
    * `ls -al build/static/js/*.js | wc`
        files 0 = 173 - 173
3. `yarn start` in ui/apps/platform

#### Manual testing in Compliance

1. Visit /main/compliance, and then click **Scan environment**.

    * Before changes, see links in graphs that **do not** have PatternFly style.
        compliance_without_PatternFly_style

    * After changes, see links in graphs that have PatternFly style.
        compliance_with_PatternFly_style

2. Click a standard link.

    * Before changes, see links in table that **do not** have PatternFly style.
        compliance_controls_without_PatternFly_style

    * After changes, see links in table that have PatternFly style.
        compliance_controls_with_PatternFly_style

3. Go back, and then click namespace count link.

    * Before changes, see links in table that **do not** have PatternFly style.
        compliance_namespaces_without_PatternFly_style

    * After changes, see links in table that have PatternFly style.
        compliance_namespaces_with_PatternFly_style

#### Manual testing in Vulnerability Management

1. Visit /main/vulnerability-management

    * Before changes, see links in graphs that **do not** have PatternFly style.
        vulnerability-management_without_PatternFly_style

    * After changes, see links in graphs that have PatternFly style.
        vulnerability-management_NumberedList_with_PatternFly_style
        vulnerability-management_NumberedGrid_with_PatternFly_style

2. Click images count link.

    * Before changes, see links in table that **do not** have PatternFly style.
        vulnerability-management_images_without_PatternFly_style

    * After changes, see links in table that have PatternFly style.
        vulnerability-management_images_with_PatternFly_style

#### Manual testing in Configuration Management

1. Visit /main/configmanagement.

    * Before changes, see links in graphs that **do not** have PatternFly style.
        configmanagement_without_PatternFly_style

    * After changes, see links in graphs that have PatternFly style.
        configmanagement_with_PatternFly_style

2. Click a controls link.

    * Before changes, see links in table that **do not** have PatternFly style.
        configmanagement_controls_without_PatternFly_style

    * After changes, see links in table that have PatternFly style.
        configmanagement_controls_with_PatternFly_style

3. Go back, and then click **Secrets** in the **Application & Infrastructure** menu.

    * Before changes, see links in table that **do not** have PatternFly style.
        configmanagement_secrets_without_PatternFly_style

    * After changes, see links in table that have PatternFly style.
        configmanagement_secrets_with_PatternFly_style

#### Manual testing in Risk

1. Visit /main/risk.

    * Before changes, see links in table that **do not** have PatternFly style.
        risk_without_PatternFly_style

    * After changes, see links in table that have PatternFly style.
        risk_with_PatternFly_style

#### Manual testing in Clusters

1. Visit /main/clusters.

    * Before changes, see links in table that have PatternFly style.
        clusters_with_PatternFly_style

#### Integration testing

* compliance/complianceDashboard.test.js
* compliance/complianceList.test.js
* configmanagement/ciscontrols.test.js
* configmanagement/clusters.test.js
* configmanagement/dashboard.test.js
* configmanagement/deployments.test.js
* configmanagement/images.test.js
* configmanagement/namespaces.test.js
* configmanagement/nodes.test.js
* configmanagement/policies.test.js
* configmanagement/roles.test.js
* configmanagement/secrets.test.js observe fail first and then fix selector
* configmanagement/serviceaccount.test.js
* configmanagement/usersandgroups.test.js
* risk/risk.test.js
* vulnmanagement/clusterCves.test.js
* vulnmanagement/clusters.test.js
* vulnmanagement/dashboard.test.js
* vulnmanagement/dashboardToEntityPage.test.js observe fail first and then fix selector
* vulnmanagement/deployments.test.js
* vulnmanagement/entitypages.test.js
* vulnmanagement/imageComponents.test.js
* vulnmanagement/imageCves.test.js
* vulnmanagement/images.test.js
* vulnmanagement/namespaces.test.js
* vulnmanagement/nodeComponents.test.js
* vulnmanagement/nodeCves.test.js
* vulnmanagement/nodes.test.js

#### Unit testing

* src/Components/TableCellLink/TableCellLink.test.js
* src/Containers/Compliance/List/tableColumns.test.js
* src/Containers/VulnMgmt/List/Cves/ListCVEs.utils.test.js
* src/Containers/ConfigManagement/ConfigMgmt.utils.test.js

### CODE REVIEW

File: ui/apps/platform/cypress/integration/vulnmanagement/entitypages.test.js [hunk 1]
```diff
@@ -54,151 +54,6 @@ describe('Entities single views', () => {
             });
     });
 
-    // ROX-15888 ROX-15985: skip until decision whether valid to assume high severity violations.
```
Comment (dvail) [inline ui/apps/platform/cypress/integration/vulnmanagement/entitypages.test.js:57]: A fair decision given our experience with this in the past.

File: ui/apps/platform/src/Components/visuals/SunburstDetailSection.js [hunk 2]
```diff
@@ -79,28 +78,19 @@ class SunburstDetailSection extends Component {
...
-                                    style={color ? { color } : null}
-                                >
+                                <span className="flex w-full word-break leading-tight">
                                     <Truncate lines={4} ellipsis={<>...</>}>
                                         {!link && text}
                                     </Truncate>
                                 </span>
-                                {selectedDatum && units !== 'percentage' && (
-                                    <span style={{ color: labelColor }}>{labelValue}</span>
-                                )}
```
Comment (dvail) [inline ui/apps/platform/src/Components/visuals/SunburstDetailSection.js:103]: Was this an obsolete prop?

**Review by dvail — APPROVED** [review]
LGTM - a large number of changed files did not translate to a large number of comments. 👍

File: ui/apps/platform/src/Components/visuals/SunburstDetailSection.js (hunk 2 above)
Comment (pedrottimark) [reply ui/apps/platform/src/Components/visuals/SunburstDetailSection.js:103]: Good question. In the 4 components that render `Sunburst` element:
* `labelValue` property has no occurrences.
* `labelColor` property has occurrence only in `CvesByCvssScore` component.

File: ui/apps/platform/cypress/integration/vulnmanagement/entitypages.test.js (hunk 1 above)
Comment (pedrottimark) [reply ui/apps/platform/cypress/integration/vulnmanagement/entitypages.test.js:57]: Yeah, and I noticed that the `skip` tests are for policies, which has been deleted from Vulnerability Management.

**Pull request merged**